src/04_printing.py

```diff
@@ -11,7 +11,8 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print("x is %d, y is %.2f, z is %s" % (x, y, z))
```

Comment:
Good work! just remember to add in the quotes for "I like turtles" by using escape characters, `\"%s\"`